sql/core/src/test/scala/org/apache/spark/sql/expressions/ExpressionInfoSuite.scala

```diff
@@ -151,6 +151,9 @@ class ExpressionInfoSuite extends SparkFunSuite with SharedSparkSession {
       "org.apache.spark.sql.catalyst.expressions.CurrentTimestamp",
       "org.apache.spark.sql.catalyst.expressions.CurrentTimeZone",
       "org.apache.spark.sql.catalyst.expressions.Now",
+      "org.apache.spark.sql.catalyst.expressions.UnixSeconds",
+      "org.apache.spark.sql.catalyst.expressions.UnixMillis",
+      "org.apache.spark.sql.catalyst.expressions.UnixMicros",
```

Comment:
why can't we run the example SQL for them?

Reply:
Originally, the timestamp creation in the examples will be affected by the local timezone
```
_FUNC_(TIMESTAMP('1970-01-01 00:00:01'))
```

Now I update the example and add the timezone in the timestamp literal. We can test examples.

Reply:
Sounds better.